src/vcpkg/base/json.cpp

```diff
@@ -995,6 +995,14 @@ namespace vcpkg::Json
             {
                 StatsTimer t(g_json_parsing_stats);
 
+                if (Strings::starts_with(origin, "\xEF\xBB\xBF"))
+                {
+                    // remove byte-order mark from the beginning of the string
+                    // this is a workaround for our localization software giving us files with a BOM
+                    // TODO: figure out how to fix the loc software
```

Comment:
```suggestion
                    // remove byte-order mark from the beginning of the string
```

I think we should just always be doing this. BOMs to mark the file as UTF-8 rather than another legacy codepage are common and reasonable.

Alternately, perhaps it should be done in `read_lines` / `read_content`?

Reply:
yeah, putting it in `read_contents` makes sense.